wakuv2/nwaku.go

```diff
@@ -3142,3 +3146,11 @@ func (n *WakuNode) GetNumConnectedPeers() (int, error) {
 	}
 	return len(peers), nil
 }
+
+func getContextTimeoutMilliseconds(ctx context.Context) int {
+	deadline, ok := ctx.Deadline()
+	if ok {
+		return int(time.Until(deadline).Milliseconds())
+	}
+	return 0
```

Comment:
I wonder if we should make the app to crash earlier. Maybe adding a log before this line and/or returning -1 may help

Reply:
ah! the 0 is not an invalid error! it means that it's a context without deadline, and a timeoutMs of 0 is equal to not having timeout!